### DISCUSSION TITLE
The trick play option on rtsp server

### DISCUSSION THREAD

**Discussion by geohuz** [bluenviron/mediamtx#2114] [Questions]
The trick play option on rtsp server
RTSP servers may optionally support 'trick play' operations on a stream - specifically, the ability to seek to a specific time within the stream, and/or the ability to play the stream in fast-forward, slow, or reverse play. This is useful for connecting NVRs playback service with recorded content which published on rtsp. More detail: http://www.live555.com/openRTSP/#trick-play.

**Comment by aler9** [comment]
Hello, this is already implemented, you can use `rtspRangeType` and `rtspRangeStart` to seek a stream:

https://github.com/bluenviron/mediamtx/blob/e4bd1b35a2b822df166c837979b0602ee52a6a92/mediamtx.yml#L310-L320

Please use issues, not discussions, to request new features.

[a comment by an automated account is not shown]
